kuma/javascript/src/article.jsx

```diff
@@ -27,12 +27,10 @@ function TranslationStatus({
     } else if (translationStatus === 'outdated') {
         content = (
             <>
-                {gettext(
-                    'You’re reading the English version of this content since no translation exists yet for this locale.'
-                )}
+                {gettext('This translation is incomplete.')}
                 &nbsp;
                 <a href={translateURL} rel="nofollow">
-                    {gettext('Help us translate this article!')}
+                    {gettext('Please help translate this article from English')}
```

Comment:
I wonder if we should mention English here. Was that not part of the confusion reported in the issue? Perhaps we can just say `Please help translate this article`?

Reply:
These articles are translate completely. Yes there are mistakes in them ,but to understand what it is possible.
Problem

Reply:
Ah, I see @sirartemis, so the problem is not the text of the message, it is that the message is shown on a page that has already been translated?

Reply:
Exactly. Whole problem in URL's. Need to change URL paths to be on English language only.
In places where main sections on Russian, no messages, but there on English - there are messages.
Need change something in code, so that there is no translation check on the translation path and do another check.
Messages appear because of this check

Reply:
> I wonder if we should mention English here. Was that not part of the confusion reported in the issue? Perhaps we can just say `Please help translate this article`?

From my understanding, translations always come from english, which is why this string was also used originally (I got both strings from the old code, just made a mistake when picking the string).

> Ah, I see @sirartemis, so the problem is not the text of the message, it is that the message is shown on a page that has already been translated?

The message is shown when a translated revision is older than 10 days and tagged as inprogress. I can't speak to how reasonable that model is, but it shouldn't have changed from the old version.